src/Workspaces/Core/Portable/Workspace/Solution/SolutionCompilationState.CreationPolicy.cs

```diff
@@ -57,6 +58,6 @@ private readonly record struct CreationPolicy(
         /// Do not create up to date source generator docs and do not create up to date skeleton references for P2P
         /// references.  For both, use whatever has been generated most recently.
         /// </summary>
-        public static readonly CreationPolicy DoNotCreate = new(GeneratedDocumentCreationPolicy.DoNotCreate, SkeletonReferenceCreationPolicy.DoNotCreate);
+        public static readonly CreationPolicy DoNotCreate = new(GeneratedDocumentCreationPolicy.CreateRequired, SkeletonReferenceCreationPolicy.DoNotCreate);
```

Comment:
nit: the parameter was requiredDocumentsOnly.  perhaps .CreateRequiredOnly.  I'm liking the 'only' to indicate very specifically exactly what it is controlling, and waht it isn't.

Reply:
Ok.  As vebose as it might be, i think `CreationPolicy CreateRequiredGeneratedDocuments_DoNotCreateSkeletonReferences` is likely appropriate.  Having this be called 'DoNotCreate' now feels misleading.